packages/components/src/ResourcePicker/Toolbar/SortOptions/SortOptions.scss

```diff
@@ -0,0 +1,10 @@
+.sort-options {
+	> * {
+		display: inline-block;
+		height: 100%;
+	}
```

Comment:
If you set 
```css
.sort-options {
	display: flex;
	align-items: center;
}
```
You can avoid styling every child, and having the label box placed weirdly to align the text